EvidenceApi/V1/UseCase/EvidenceRequestValidator.cs

```diff
@@ -38,9 +38,16 @@ public EvidenceRequestValidator(IValidator<ResidentRequest> residentValidator, I
             RuleForEach(x => x.DeliveryMethods)
                 .IsEnumName(typeof(DeliveryMethod), false);
 
+
```

Comment:
It's probably best to use a single empty line whenever one is needed readability.